compose.yaml

```diff
@@ -0,0 +1,18 @@
+services:
+  proxy:
+    image: nginx
```

Comment:
The `image` field makes Docker Compose consider the `nginx` image to be equivalent to whatever will be built by the `build` section so it can end up erroneously using that instead of building the image.
```suggestion
```

We could specify our own name here (sth like `ghcr.io/YetAnotherSpieskowcy/travel-agency-app-proxy:latest`) but since there's no need for it, I would suggest that we just remove the field entirely.